lib/streamlit/DeltaGenerator.py

```diff
@@ -804,6 +806,14 @@ def exception(self, element, exception, exception_traceback=None):
 
         exception_proto.marshall(element.exception, exception, exception_traceback)
 
+    @_with_element
+    def arrow_table(self, element, df):
+        import streamlit.elements.arrow_table as arrow_table
+
+        element_path = (self._container,) + self._path + (self._id,)
+        default_uuid = hash(element_path)
```

Comment:
I don't like this part - it means that if the element order changes, the table will be assigned a new uuid and needs to be redrawn.

Reply:
Just add a comment explaining that this will bust the message cache and we should come up with an improvement